src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -268,33 +269,100 @@ class StreamGroup extends React.Component<Props, State> {
         {hasGuideAnchor && <GuideAnchor target="issue_stream" />}
         {withChart && (
           <ChartWrapper className="hidden-xs hidden-sm">
-            <GroupChart
-              statsPeriod={statsPeriod!}
-              data={data}
-              showSecondaryPoints={showSecondaryPoints}
-            />
+            {!data.filtered?.stats && !data.stats ? (
+              <Placeholder height="24px" />
+            ) : (
+              <GroupChart
+                statsPeriod={statsPeriod!}
+                data={data}
+                showSecondaryPoints={showSecondaryPoints}
+              />
+            )}
           </ChartWrapper>
         )}
         <EventUserWrapper>
-          <DropdownMenu isNestedDropdown>
-            {({isOpen, getRootProps, getActorProps, getMenuProps}) => {
-              const topLevelCx = classNames('dropdown', {
-                'anchor-middle': true,
-                open: isOpen,
-              });
-
-              return (
-                <GuideAnchor target="dynamic_counts" disabled={!hasGuideAnchor}>
+          {!defined(primaryCount) ? (
```

Comment:
our dropdowns being rendered all the time makes me feel some type of way

Reply:
You could not render them using `isOpen` (unless you need animations)